src/write_gcm.c

```diff
@@ -2147,6 +2176,53 @@ static monitored_resource_t *wg_monitored_resource_create(
   return NULL;
 }
 
+static monitored_resource_t *wg_monitored_resource_create_from_metadata(
```

Comment:
`..._from_metadata_agent`. The other `wg_monitored_resource_create_...` functions also create it from metadata (server, in those cases).